examples/liquid.ini

```diff
@@ -277,6 +308,26 @@ image_classification_object_detection_enabled = false
 
 
 
+;[nlp_service]
```

Comment:
Let's not add a new section for this. I see it's not read in the configuration file, so this is left over dead code i suppose.